ui/src/views/mint/form-step/mint-form.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button, Card, Grid, Icon, IconButton, Stepper } from '@/components';
+import { Mint } from '../mint.context';
+import { LogoField } from './fields/logo/logo-field';
+import { AppDescriptionField, AppNameField, EnsDomainField } from './fields';
+import { VerifyNFAField } from './fields/verify-nfa-field';
+import { MintCardHeader } from '../mint-card';
+
+export const FormStep = () => {
+  const { prevStep, nextStep } = Stepper.useContext();
+  const { appName, appDescription, domain } = Mint.useContext();
+
+  return (
+    <Card.Container css={{ width: '$107h' }}>
+      <MintCardHeader title="NFA Details" onClickBack={prevStep} />
+      <Card.Body>
+        <Grid css={{ rowGap: '$6' }}>
+          <Grid css={{ rowGap: '$4' }}>

```

Comment:
These elements could be a single grid using grid template and areas.

Reply:
will do!